foundations/javascript_basics/fundamentals-5.md

```diff
@@ -21,18 +21,18 @@ Besides being a quick and handy way to store data, arrays also have a set of fun
 1. Start out by watching [this video](https://www.youtube.com/watch?v=HB1ZC7czKRs) from Wes Bos.  To follow along, use your local Javascript30 repository. If you don't have the repo yet, clone it from [here](https://github.com/wesbos/JavaScript30).
 2. Watch and code with [Part 2](https://www.youtube.com/watch?v=QNmRfyNg1lw) of Wes Bos's array series.
 
-### Practice
+### Assignment
 
 You have been through a _ton_ of content, and it would not be surprising if you were feeling rather overwhelmed at this point.  Let's take a little time to slow down and practice this stuff with more coding exercises.  I'll warn you now: some of these get pretty difficult, but if you've done the lessons you should have covered everything you need to know to solve them.  Don't be afraid to go back and review if you find that you can't remember something, that's part of the process.  If you get stuck come join us in the chatroom!
 
 <div class="lesson-content__panel" markdown="1">
 At this point you just need a little more practice!  Go back to the [JavaScript exercises](https://github.com/TheOdinProject/javascript-exercises) that we introduced in the Fundamentals 4 project and complete the following:
 
-- calculator
-- palindromes
-- fibonacci
-- getTheTitles
-- findTheOldest
+* calculator
+* palindromes
+* fibonacci
+* getTheTitles
+* findTheOldest
 
 Don't remember how to setup and test the exercises? No worries! Check out the [Fundamentals Part 4 lesson](https://www.theodinproject.com/lessons/foundations-fundamentals-part-4#practice) for a refresher.
```

Comment:
For this line, let's add in at the end something from the line above being removed:

```suggestion
Don't remember how to setup and test the exercises? No worries! Check out the [Fundamentals Part 4 lesson](https://www.theodinproject.com/lessons/foundations-fundamentals-part-4#practice) for a refresher. If you feel yourself getting overwhelmed or stuck, don't be afraid to go back and review or ask for help on our Discord!
```

Reply:
Done. https://github.com/TheOdinProject/curriculum/pull/24248/commits/27c91f1cef63202ae0ab7c0f79913ab0a8b19841